ISSUETEMPLATE.md

```diff
@@ -0,0 +1,30 @@
+## LGT GridIron Project
+
+## Issue Title:
+[Concisely summarize the issue or feature request in a few words]
+
+## Description:
+[Provide an overview of the issue or feature request, including relevant context or background information that helps in understanding the problem. Additionally, include any pertinent details such as workaround solutions, related issues, etc.]
+
+## Steps to Reproduce (for bugs):
+[If applicable, provide a clear set of steps to reproduce the issue. This helps developers understand and fix the problem more effectively.]
+
+## Expected Behavior:
+[Describe what you expected to happen when performing the steps above.]
+
+## Actual Behavior:
+[Describe what happens when performing the steps above.]
+
+## Screenshots / Logs (if applicable):
+[Attach any relevant screenshots or log files that help illustrate the issue.]
+
+## Environment:
```

Comment:
No need for this. This will be overkill for now.

Reply:
removed unnecessary ##environment section